src/test/java/com/witherview/selfPractice/SelfQuestionApiTest.java

```diff
@@ -15,13 +21,19 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
-public class SelfQuestionApiTest extends MockMvcSupporter {
-    final Long id = (long) 5;
-    final String question = "당신의 주 언어는 무엇인가요?";
-    final String answer = "저의 주 언어는 ~~입니다";
-    final Integer order = 1;
-    final String updatedQuestion = "당신의 입사 후 목표는 무엇인가요?";
-    final String updatedAnswer = "저의 목표는 ~입니다.";
+public class SelfQuestionApiTest extends SelfPracticeSupporter {
+
+    @Autowired
+    QuestionRepository questionRepository;
+
+    @BeforeEach @Transactional
+    public void 미리질문등록() {
+        QuestionList questionList = questionListRepository.findById(listId)
+                .orElseThrow(() -> new NotFoundQuestionList());
+        Question question = new Question("question", "answer", 1);
+        questionList.addQuestion(question);
+        questionRepository.save(question);
+    }
```

Comment:
다음과 같이 하나의 메소드 안에다가 작성하시고 테스트 클래스에다가 Transactional을 붙이시면 해결 되실 것 같아요~

안되던 이유는 음.. 더 찾아봐야 할 것 같은데 추정하기로는
1. BeforeEach가 어떤 것 부터 시작되는지가 보장이 안됨 (틀릴 수 있습니다)
2. 하나의 트랜잭션 안에서 테스트 전 로직이 전부 실행되지 않아 영속성 문제가 발생(유력)

`listId`, `questionId`가 고정되어 있어서 테스트가 실패하는게 있네요~ :D

```suggestion
@Transactional
public class SelfQuestionApiTest extends SelfPracticeSupporter {

    @Autowired
    QuestionRepository questionRepository;

    @BeforeEach
    public void 미리질문등록() {
        User user = userRepository.findByEmail(email);
        if (user == null) {
            user = userRepository.save(new User(email, passwordEncoder.encode(password), name));
        }
        AccountSession accountSession = new AccountSession(userId, email, name);
        mockHttpSession.setAttribute("user", accountSession);

        QuestionList questionList = new QuestionList("title", "enterprise", "job");
        user.addQuestionList(questionList);
        questionListRepository.save(questionList);

        Question question = new Question("question", "answer", 1);
        questionList.addQuestion(question);
        questionRepository.save(question);
    }
```